### PR TITLE
MSP-4424 No more separate StartBatch/AddToBatch api calls. They have been merged towards ExecuteBatch api call.

### PR DESCRIPTION
**Pull request by marinhekman**

### CODE REVIEW

File: Assets/Scripts/Data Model/GameSession.cs [hunk 1]
```diff
@@ -12,7 +12,6 @@ public enum SessionState { Request, Initializing, Healthy, Failed, Archived }
 
 		public int id;
 		public string name;
-		public int game_config_version_id;
```
Comment (marinhekman) [inline Assets/Scripts/Data Model/GameSession.cs:15]: see 
https://github.com/BredaUniversityResearch/MSPChallenge-Client/pull/116

File: Assets/Scripts/Networking/BatchRequest.cs [hunk 2]
```diff
@@ -178,42 +87,30 @@ public void ExecuteBatch(Action<BatchRequest> a_successCallback, Action<BatchReq
...
-				//All add requests are in, execute the batch
-				m_executeWhenReady = false;
-				m_status = EBatchStatus.AwaitingResults;
-
-				NetworkForm form = new NetworkForm();
-				form.AddField("batch_id", m_batchID);
-				form.AddField("async", m_async.ToString());
+			NetworkForm form = new NetworkForm();
+			form.AddField("country_id", SessionManager.Instance.CurrentUserTeamID);
+			form.AddField("user_id", SessionManager.Instance.CurrentSessionID);
```
Comment (marinhekman) [inline Assets/Scripts/Networking/BatchRequest.cs:95]: These fields are from the previous "StartToBatch" api call

File: Assets/Scripts/Networking/BatchRequest.cs [hunk 3]
```diff
@@ -178,42 +87,30 @@ public void ExecuteBatch(Action<BatchRequest> a_successCallback, Action<BatchReq
...
+			form.AddField("batch_guid", m_batchGuid.ToString());
```
Comment (marinhekman) [inline Assets/Scripts/Networking/BatchRequest.cs:96]: The batch guid is added. and necessary to "back reference" the correct batch when the ExecuteBatch response is given, which has this guid

File: Assets/Scripts/Networking/BatchRequest.cs [hunk 4]
```diff
@@ -95,16 +62,7 @@ public int AddRequest(string a_endPoint, JObject a_data, int a_group)
 				return -1;
```
Comment (HutchinsonKevin) [inline Assets/Scripts/Networking/BatchRequest.cs:62]: This should probably fail when not in the Setup phase right?

Comment (HutchinsonKevin) [reply Assets/Scripts/Networking/BatchRequest.cs:62]: Same below

**Review by HutchinsonKevin — APPROVED** [review]

**Review by hwarmelink — APPROVED** [review]

**Pull request merged**